types/wx-js-sdk-dt/index.d.ts

```diff
@@ -0,0 +1,468 @@
+// Type definitions for wx-js-sdk 1.2
+// Project: https://res.wx.qq.com/open/js/jweixin-1.2.0.js
+// Definitions by: Bian Zhongjie <https://github.com/agasbzj>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export namespace wx {
```

Comment:
Sorry, `reference` is for external definitions, there's no need to use `reference` here.